server/views/components/promo/index.njk

```diff
@@ -1,14 +1,64 @@
-<div class="promo">
-  <img class="promo__image"
-    src="{{ image.sources[(image.sources | length) - 1].src }}"
-      srcset="{%- for source in image.sources %}
-        {{ source.src }} {{ source.size }}w
-        {%- if (source !== (image.sources | last)) %}, {%- endif %}
-      {%- endfor %}"
-      alt="{{ image.alt }}" />
+{% set gallery = 'actions/gallery-link' %}
```

Comment:
Why not just have these in the code?
Ot name them `galleryIcon` to make more sense?